### ISSUE TITLE
Cannot link notebooks in subdir

### ISSUE THREAD

**Issue by ycopin** [spatialaudio/nbsphinx#199]
Thanks for your tool, but I have difficulties to add links to notebooks in sub-directories (even though it's supposedly possible from doc). I created a minimal example (https://www.ipnl.in2p3.fr/perso/ycopin/_static/documents/Test.tgz):
```
Test
├── _build
├── conf.py
├── index.rst
├── Makefile
├── _static
├── Subdir
│   ├── intro.rst
│   └── notebook.ipynb
└── _templates
```
In `intro.rst`, I try the 2 suggested links:
```
:ref:`reflink <Subdir/notebook.ipynb>` or `link <Subdir/notebook.ipynb>`_
```
but none of them work. The 1st link is actually inactive, and the 2nd links to doubly-inexistent `_build/html/Subdir/Subdir/notebook.ipynb` (note the `Subdir/Subdir` and the `ipynb` extension). 

Cheers,

**Comment by mgeier** [comment]
The two ways to create those links are quite different.

When you use Sphinx hyperlinks (https://nbsphinx.readthedocs.io/en/0.3.3/a-normal-rst-file.html#links-to-notebooks-and-other-sphinx-source-files), you have to specify a *relative* path.

When you use `:ref:` links (https://nbsphinx.readthedocs.io/en/0.3.3/a-normal-rst-file.html#links-to-notebooks-ye-olde-way), you have to specify an *absolute* path that starts with a slash (latter is a breaking change introduced in version 0.3, see #136).

Therefore, it should work with

    :ref:`reflink </Subdir/notebook.ipynb>` or `link <notebook.ipynb>`_

**Comment by ycopin** [comment]
Thanks for the feedback! The 1st link is indeed working now, but the second is not. It raises:
```
/home/ycopin/tmp/Test/Subdir/intro.rst:4: WARNING: unknown document: /subdir/notebook
```
at compilation time, and the link is inactive in HTML output. 

Removing the `ipynb` extension (i.e. `link <notebook>`) solves the compilation issue and creates the link, but it points to inexistent file `file:///home/ycopin/tmp/Test/_build/html/Subdir/notebook` (without `html` extension).

**Comment by mgeier** [comment]
OK, I see, this is a bug: #196.
I've fixed this bug in #198 and I've just created a new release.
If you update `nbsphinx` now, it should hopefully work!

**Comment by ycopin** [comment]
Great! Working fine w/ nbsphinx-0.3.4. Thanks for the support.
